changelog/1.17.0.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "1.17.0"
+description: "Released on 03/17/2021"
+---
+
+### Breaking Changes ❗
+
+- (?) Wsp orgs migration
+
+### Features ✨
+
+- Environment Providers **TODO**
+- Autostart: Autostart allows you to set the time when Coder
+  automatically starts and builds your environments.
+  - web: A new onboarding wizard is available for setting a preferred Autostart
+    time.
+  - web: Visit your account preferences > Autostart to set a time and any
+  environments that should be automatically started. We recommend enabling this
+  feature for the environments you use the most.
+  - web: Autostart action has been added to the Audit Log
+  - web: A new checkbox for opting into Autostart is available on the Create and
+    Edit Environment modals.
+
+- web: Environment stop actions have been added to the Audit Log
+- web: Show most recent Build Log for offline environments
+- cli: Improved sign in flow with (cdr/coder-cli#274)
+- web: Improved error messages within the Build Log
+  - Improve messages when '/coder/configure' is not executable
+- code-web: Upgrade to code-server version 3.9.0
+- jetbrains: The Mac OS keymap plugin is now automatically installed
+- web: Resources section on Environments page has been redesigned
+- web: Apply consistent design theme across all Preferences pages
+
+### Bug Fixes 🐛
+
+- web: Fix DevURL error page links using incorrect URIs.
+- infra: Fix timeout when deleting users with many environments
+- infra: Increase offline timeout in trial versions of Coder
+- web: Use proper casing when referencing 'GitHub' and 'GitLab'
+- infra: Fix renaming an OAuth provider without creating a duplicate. Migrating
+  to 1.17.0 will remove these duplicates. They are not visible in the UI.
+- infra: Correct environment statuses during build and shutdown states
+- web: Fix SiteAdmin users receiving a notification that their SSH keys are out
+  of sync after creating an initial environment.
+  - Note: We do not recommend using the SiteAdmin user beyond initial setup
+
+### Security Updates 🔐
+
+- Resetting the SiteAdmin password will now always be marked as temporary and
+  require a new one to be set upon sign in.
```

Comment:
cc @jawnsy to verify

Reply:
this is true for `cemanager reset-admin-password`, but the changelog entry is ambiguous here (i.e. if you reset via the UI or API then this is not true)

Reply:
Agreed, I wasn't sure on scope. Some users `kube exec` that command I believe - do you know?

Reply:
sorry I phrased my comment badly, suggest we use this in the changelog instead (I'd have done this as a suggestion but I still have no idea how to mark those up correctly):

```
- When resetting the SiteAdmin password using the
  `cemanager reset-admin-password` tool, Coder now requires a new
  password to be set upon next login (it is marked as a temporary
  password).
```

Reply:
That's awesome ty !

Reply:
The parenthetical part is optional, depending o whether we want to talk about the implementation detail of the temporary password flag

Reply:
https://github.com/cdr/docs/pull/166/commits/57f04f959dcd596a078f810e3e89fab915586d20